GOVERNANCE.md

```diff
@@ -25,4 +26,25 @@ Contributors are encouraged to ask for collaborator access once you have had a f
 
 ## Process for releasing a new version of Wikimate
 
-[TODO]
+A PR should be created with all relevant changes to update the repository for the upcoming release.
+(See [#81](https://github.com/hamstar/Wikimate/pull/81) for an example.)
+It should apply the following actions:
+
+1. Change the "Unreleased" heading in the CHANGELOG.md file
```

Comment:
> [Discussion](https://github.com/hamstar/Wikimate/pull/45#issuecomment-252443635) led to issue #48, and the resulting commits were approved without further discussion it seems. So I guess our agreement was more implicit than I remembered.

Ah, yes, thanks for the links :)

> Does adhering to a standard mean that we have no creative freedom to deviate in how we word something? [Precedent](https://gcc.gnu.org/wiki/GFortran/News#gfortran_7_.28current_development_version.29) [does](http://docs.nagvis.org/1.3/en_US/changelog.html) [exist](http://pyhrf.org/changelog.html), although I've not found many more examples.

I'd say we can deviate from the standard, if sufficient motive exists.

> Anyway, I like "current development" better because it has a more positive and active connotation. Many words starting with "un" feel less so.

I can agree with this argument, but I'd rather we picked a different section name that still conforms to it. I'm afraid "Current development" may not be sufficiently clear. It certainly feels slightly awkward to my (non-native English speaker) ears, for the meaning it is intended to have.

> How exactly do you mean "fits better with the other headings"?

I meant that all headings are version this, version that; so it would logically make sense (IMO) that the top heading is "upcoming version", "next release", "unreleased [version]", or something to that effect. Do any of these (or a variation thereof) appeal to you?